Assignment2/u120324.ttl

```diff
@@ -0,0 +1,13 @@
+@prefix vocab: <http://www.ontologies.org/ontologies/LuisGaldo#>.
+@prefix : <http://www.ontologies.org/ontologies/LuisGaldo#> .
+
+:Class01 vocab:includes:Computer101,Sensor029.
+
+:Sensor029 vocab:hasMeasurement:Measurement8401.
+
+:Measurement8401 vocab:hasTemperature:29;
```

Comment:
Look at other files that have been already accepted. The literal values have to be presented as typed literals. Same for dates